corehq/apps/domain/project_access/models.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from django.db import models
+
+ENTRY_RECORD_FREQUENCY = datetime.timedelta(hours=6)
+
+
+class SuperuserProjectEntryRecord(models.Model):
```

Comment:
This is probably going to end up being a long list as HQ grows. should we partition this table now? (maybe by week or month?) http://architect.readthedocs.io/features/partition/postgresql.html